loops-and-arrays.js

```diff
@@ -42,6 +64,14 @@ const numbers = [2, 4, 5, 29, 38];
 
 const squares = [30, 2, 8, 24, 11];
 
+console.log(squares);
+
+for (let i = 0; i < squares.length; i++){
+    const volume = squares [i] * squares [i] *squares [i];
+    squares [i] = "Het volume van " + squares [i] + " is " + volume;
+}
+
```

Comment:
Netjes gedaan! Het wordt wel een probleem wanneer je een nummer tot de 15e macht hebt... Dan zou je 15x ...* squares[i] moeten invullen. Dan is het handiger om de juiste operator te gebruiken. Kortste weg principe!